src/Serializers/FingerprintSerializer.php

```diff
@@ -57,40 +71,16 @@ private function addIdToSerialization( EntityId $id = null, array &$serializatio
 		$serialization['id'] = $id->getSerialization();
 	}
 
-	private function addLabelsToSerialization( TermList $labels, array &$serialization ) {
-		$serialization['labels'] = $this->serializeValuePerTermList( $labels );
-	}
-
+	/**
+	 * @deprecated this is used somewhere stupid...
```

Comment:
Yeah indeed, this is used by some tests. I ran into this when I was cleaning up some stuff in this component recently. I do not think the deprecation tag makes a lot of sense though, since this whole thing is package private.